mojo/stdlib/test/memory/test_span.mojo

```diff
@@ -293,7 +293,7 @@ def test_apply():
         return x * 2
 
     @parameter
-    fn _where[D: DType, w: Int](x: SIMD[D, w]) -> SIMD[DType.bool, w]:
+    fn _is_even[D: DType, w: Int](x: SIMD[D, w]) -> SIMD[DType.bool, w]:
```

Comment:
I *think* the more "pythonic" name would be simply even? Again, I'm fine with either.